docs/GettingStarted.md

```diff
@@ -11,19 +11,30 @@ To start with, you'll want to download the .NET 7 SDK (https://dotnet.microsoft.
 All other code and library dependencies are handled by NuGet and will be downloaded automatically when the application is built.
 There are no other language or runtime dependencies.
 
+### Useful tools:
+Visual Studio and JetBrains Rider both have avalonia extensions that allow previewing ui designs (.axaml files).
+- Visual Studio:  https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaVS
+- Rider: https://plugins.jetbrains.com/plugin/14839-avaloniarider
+
+Both extensions can be found directly from the IDE plugin/extension managers.
+Using these extension is recommended for easier development.
+
+There are also some project Templates for Avalonia that can be very useful to have, available here:
```

Comment:
Looks good to me. Might want to change this to also mention `Item Templates`, since said repo also contains templates for items (`Window`s, `UserControl`, `ResourceDictionary` etc.). 

While it's unlikely anyone working on this would would create a new project; they are likely to make a new item from template.